tutorials/hydra.md

```diff
@@ -0,0 +1,339 @@
+
+9 minut czytania
+
+Ten post jest przeznaczony jako krótkie, samodzielne wprowadzenie do narzędzia Hydra. Omówiono w nim wiele tematów, w tym jak tworzyć obiekty klas, uruchamiać przeszukiwanie parametrów i sprawdzać poprawność konfiguracji w czasie pracy. Wprowadzenie nie obejmuje pełnego zakresu opcji oferowanych przez Hydrę. W tym celu należy zgłębić [oryginalną dokumentację](https://hydra.cc/docs/next/intro).
+
+> 🔗 Kod dla tego tutoriala jest dostępny na repozytorium GitHub: [https://github.com/sscardapane/hydra-tutorial](https://github.com/sscardapane/hydra-tutorial).
+
+Instalacja i przegląd
+-------------------------
+
+Będziemy korzystać z wersji 1.1 biblioteki Hydra, którą można zainstalować poprzez:
+
+    pip install hydra-core==1.1.0.rc1
+    
+
+> ⚠️ Uwaga: kilka poniższych instrukcji nie będzie działać poprawnie na poprzednich wersjach biblioteki. Wszystkie zmiany pomiędzy wersjami są udokumentowane [na stronie twórców](https://hydra.cc/docs/next/upgrades/1.0_to_1.1/changes_to_hydra_main_config_path).
+
+Część kodu oparta jest na [PyTorch](https://pytorch.org/),ale można go łatwo dostosować do innych frameworków głębokiego uczenia. Rozważamy klasyczny scenariusz dostrajania, w którym dostrajamy mały model klasyfikacyjny na wierzchu wstępnie wytrenowanej konwencjonalnej sieci neuronowej. Nasza ostateczna konfiguracja będzie wyglądać tak (rysunek może z początku wydawać się niejasny):
+
+![Alt text](../../../../../../../../../C:/Users/User/Desktop/Matematyka%20stosowana%20-%20II%20stopie%C5%84/III%20semestr/Big%20Data%202022/BigData2022-films/tutorials/hydra.jpg)
+
+Po lewej stronie mamy szereg plików konfiguracyjnych, opisujących kolejno:
+
+1.  Konfiguracja dla zbioru danych;
+2.  Konfiguracja naszej wstępnie wytrenowanej sieci;
+3.  Konfiguracja naszego klasyfikatora wytrenowanego "na szczycie" sieci wstępnie wytrenowanej.
+
+W centrum, Hydra automatycznie ładuje i komponuje nasze pliki konfiguracyjne, dynamicznie nadpisując każdą wartość, którą zażądamy w czasie pracy. 
+
+Po prawej stronie, nasz skrypt treningowy wykorzystuje wynikowy obiekt słownikowy do budowy naszego modelu.
+
+Będziemy budować tę konfigurację krok po kroku, poznając po kolei kilka funkcjonalności Hydry. Mając to na uwadze, zaczynajmy!
+
+Pierwszy krok: manipulowanie plikiem YAML
+-------------------------------------
+
+Podczas gdy istnieje [ogromna liczba](https://github.com/vinta/awesome-python#configuration) sposobów na określenie pliku konfiguracyjnego, Hydra pracuje z plikami YAML. Zaczynamy od stworzenia prostego pliku `config.yaml` zawierającego kilka szczegółów dotyczących naszego (fałszywego) zbioru danych obrazkowych:
+
+    # config.yaml
+    dataset:
+      image:
+        size: 124
+        channels: 3
+      classes: 10
+    
+
+[OmegaConf](https://omegaconf.readthedocs.io/en/2.0_branch/) jest prostą biblioteką umożliwiającą dostęp i manipulowanie plikami konfiguracyjnymi YAML:
+
+    from omegaconf import OmegaConf
```

Comment:
Nie chodzi mi abyś to teraz poprawiał - ale nie widzę bloków markdownowych.

Reply:
Poprawione